spec/forms/flood_risk_engine/steps/local_authority_address_form_spec.rb

```diff
@@ -58,35 +40,33 @@ def form_requires_address_lookup
         }
 
         it "is valid when valid UK UPRN supplied via drop down but no street_address present" do
-          form_requires_address_no_street_lookup do
-            expect(form.validate(valid_but_no_street_address)).to eq true
-          end
+          mock_ea_address_lookup_find_by_uprn
+          expect(form.validate(valid_but_no_street_address)).to eq true
```

Comment:
perhaps this style  more appropriate

 allow_any_instance_of(AddressServices::FindByPostcode).to receive("search").and_return([])